code/transportation-development-services/tds.css

```diff
@@ -495,802 +318,374 @@ a.form-link:hover {
 /*** Styles the Reviewer Dashboard page ***/
 /******************************************/
-#kn-scene_249 .kn-detail-label {
-  background-color: transparent;
-  min-width: 0% !important;
-}
+#kn-scene_249 .kn-detail-label { background-color: transparent; min-width: 0% !important; }
 
 /******************************************/
 /****** Styles the Memo Details page ******/
 /******************************************/
-#kn-scene_254 .kn-detail-label {
-  background-color: transparent;
-  min-width: 0% !important;
-}
+#kn-scene_254 .kn-detail-label { background-color: transparent; min-width: 0% !important; }
 
 /********************************************/
 /****** Styles the Generated Memo page ******/
 /********************************************/
-#view_698 .kn-detail-label {
-  background-color: transparent;
-}
-#view_701 .kn-detail-label {
-  background-color: transparent;
-  min-width: 0% !important;
-}
-#view_704 .kn-detail-label {
-  background-color: transparent;
-  min-width: 10% !important;
-}
-#view_705 .kn-detail-label {
-  background-color: transparent;
-  min-width: 0% !important;
-}
-#view_706 .kn-detail-label {
-  background-color: transparent;
-  min-width: 10% !important;
-}
-#view_707 .kn-detail-label {
-  background-color: transparent;
-  min-width: 0% !important;
-}
-#view_710 .kn-detail-label {
-  background-color: transparent;
-  min-width: 0% !important;
+#view_698 .kn-detail-label { background-color: transparent; }
+#view_701 .kn-detail-label { background-color: transparent; min-width: 0% !important; }
+#view_704 .kn-detail-label { background-color: transparent; min-width: 10% !important; }
+#view_705 .kn-detail-label { background-color: transparent; min-width: 0% !important; }
+#view_706 .kn-detail-label { background-color: transparent; min-width: 10% !important; }
+#view_707 .kn-detail-label { background-color: transparent; min-width: 0% !important; }
+#view_710 .kn-detail-label { background-color: transparent; min-width: 0% !important; }
+#view_711 .kn-detail-label { background-color: transparent; min-width: 0% !important; }
+#view_712 .kn-detail-label { background-color: transparent; min-width: 0% !important; }
+
+/********************************************/
+/****** Prevent Memo Table Page Breaks ******/
+/********************************************/
+/*knack tables*/
+@media print {
+  .kn-table {page-break-inside: avoid;}
 }
-#view_711 .kn-detail-label {
-  background-color: transparent;
-  min-width: 0% !important;
+
+/*rich text tables*/
+@media print {
+  tables {page-break-inside: avoid;}
 }
-#view_712 .kn-detail-label {
-  background-color: transparent;
-  min-width: 0% !important;
+
+/*******************************************/
+/********* Create Memo Page Breaks *********/
+/*******************************************/
+/*knack details views*/
+@media print {
+  #view_707 {page-break-before: always;}//Proposed Conditions section
 }
+
+/*******************************************/
```

Comment:
You could probably remove this commented out code since it looks like this view is being removed in your [other PR](https://github.com/cityofaustin/atd-knack/pull/24/files)